utxo/utxo.go

```diff
@@ -784,7 +788,11 @@ func (uv *UtxoVM) sortUnconfirmedTx() (map[string]*pb.Transaction, TxGraph, erro
 	var totalDelay int64
 	now := time.Now().UnixNano()
 	for txID, tx := range txMap {
-		totalDelay += (now - tx.Timestamp)
+		txDelay := (now - tx.Timestamp)
```

Comment:
Just curious: If someone post a Tx with a fake timestamp which is ahead of current time for thousands of seconds, it the Tx will be pending in unconfirm Tx?  If so, how we can deal with such Tx using delay time?

Reply:
yes, that is a bug. I have opened another issue: #92

Reply:
I see